Assets/_Laz/Scripts/Environment/ElectricGate.cs

```diff
@@ -31,30 +31,20 @@ public void Update()
                 {
```

Comment:
_elapsedTime += Time.deltaTime;

pass in the `Time.deltaTime` in through the `Update()` argument parameter

`Update(float deltaTime)`

This way, you can actually test the `Update()` function